R/transformCounts.R

```diff
@@ -374,8 +382,10 @@ setMethod("relAbundanceCounts",signature = c(x = "SummarizedExperiment"),
 .apply_pseudocount <- function(mat, pseudocount){
     # If "pseudocount" is not FALSE, it is numeric value specified by user. 
     # Then add pseudocount.
+
     if(!pseudocount==FALSE){
         mat <- mat + pseudocount
     }
+
```

Comment:
In fact I have always thought that code is easier to read (to me at least) when there is some space. I do not see a problem in having space to improve readability but I have now removed the empty lines per this request. Curious to know if there is a justification or is this just a stylistic preference.

Reply:
It is generally regarded as a "code smell" signifying other things at play. These could be:
- Not enough compartmentalizing with functions
- A missing comment to summarize a code chunk
- Unclear coding style (short/unclear variable name and other unclear formating)

Because we don't do it generally in the rest of the project, I would like to avoid letting it spread further.

Reply:
Fine, let's do so then. I just find it less comfortable to read but I can live with that.